ubid.rb

```diff
@@ -55,6 +55,7 @@ class UBID
   TYPE_PAYMENT_METHOD = "pm"
   TYPE_GITHUB_INSTALLATION = "g1"
   TYPE_GITHUB_RUNNER = "gr"
+  TYPE_POSTGRES_SERVER = "pd"
```

Comment:
Oh noooooo `pd` because `pg` is taken? Man, I wonder if we should take this one in the teeth and reassign Page to a new shortcode given they're not really external identifiers and we're pretty early in this whole thing.

Reply:
Yep. After seeing pg is taken, I attempted to use ps as postgres server, but ps is also taken (private subnet) :smiling_face_with_tear:. Maybe this was just bad luck, but getting 2 back-to-back conflicts this soon in our life cycle made me think that 2 characters might not be enough as add more services/models.

I'm totally onboard with stealing pg from pages. I'll extract this to separate PR so that we can deploy it sooner with more control.

Reply:
I guess I was originally inspired by the FEBE postgres protocol, where *one* character was enough. My guess is "s" is pretty heavily subscribed (server) or certain other characters.

We might have to be a bit more vigilant when making these up

Reply:
I committed and deployed a change to make page shortcode to be pa.